.github/continuous-integration.yml

```diff
@@ -0,0 +1,40 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  lint:
+    name: "Lint"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-binary:
+          - "php7.4"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2.0.0"
```

Comment:
See https://github.com/actions/checkout.

Note: There currently is no proper version resolving, so I believe it's best to reference tags.

Some action authors continually update the tag - for an example, see https://github.com/shivammathur/setup-php/releases.

For reference, see https://github.community/t5/GitHub-Actions/Version-numbering-for-Actions/m-p/32282#M1070.